addons/auth_passkey/static/src/views/auth_passkey_identity_check_form_view.js

```diff
@@ -0,0 +1,37 @@
+/** @odoo-module **/
+
+import { rpc } from "@web/core/network/rpc";
+import { registry } from "@web/core/registry";
+import { FormController } from "@web/views/form/form_controller";
+import { formView } from "@web/views/form/form_view";
+import { startAuthentication } from "../../lib/simplewebauthn.js";
+
+export class PassKeyIdentityCheckFormController extends FormController {
+    setup() {
+        super.setup();
+    }
```

Comment:
This whole block can be deleted, as overriding setup to only call super().setup() is pointless.